scalecube-configuration/src/test/java/io/scalecube/configuration/CouchbaseExperimentalTest.java

```diff
@@ -0,0 +1,99 @@
+package io.scalecube.configuration;
+
+import com.couchbase.client.java.AsyncBucket;
+import com.couchbase.client.java.CouchbaseCluster;
+import com.couchbase.client.java.document.JsonDocument;
+import com.couchbase.client.java.document.json.JsonObject;
+import java.time.Duration;
+import java.util.Random;
+import java.util.UUID;
+import org.junit.Test;
+import org.junit.jupiter.api.Disabled;
+import reactor.core.publisher.Mono;
+import rx.RxReactiveStreams;
+
+class Scratch {
```

Comment:
What is the purpose of having this class?
Please use `Scratch` locally without committing it to the github repository.